patches/helium/linux/add-middle-click-paste-flag.patch

```
@@ -0,0 +1,91 @@
+--- a/chrome/browser/helium_flag_choices.h
++++ b/chrome/browser/helium_flag_choices.h
+@@ -33,6 +33,8 @@ namespace helium {
+   constexpr const char kHeliumCatUiCommandLine[] = "helium-cat-ui";
+   constexpr const char kHeliumCatFixedAddressBarCommandLine[] =
+       "helium-cat-fixed-address-bar";
++  constexpr const char kMiddleClickPasteEnabledCommandLine[] =
++      "helium-middle-click-paste-enabled";
+ 
+ }  // namespace helium
+ 
+--- a/chrome/browser/helium_flag_entries.h
++++ b/chrome/browser/helium_flag_entries.h
+@@ -36,4 +36,8 @@
+      "[CAT] Fixed address bar width",
+      "Prevents the address bar from shrinking in the CAT layout. Will push toolbar actions outside of bounds on small window sizes, beware. Works only when the main CAT flag is enabled. Early and buggy, stuff will break and/or look weird. Helium flag.",
+      kOsDesktop, FEATURE_VALUE_TYPE(features::kHeliumCatFixedAddressBar)},
++    {helium::kMiddleClickPasteEnabledCommandLine,
++     "Middle Click Paste",
++     "Enables middle click paste. Helium flag.",
```

it's enabled by default, so maybe the description is a bit misleading? not sure how to phrase it otherwise though